openedx/features/edly/context_processor.py

```diff
@@ -50,35 +48,39 @@ def edly_app_context(request):  # pylint: disable=unused-argument
     """
     Context processor responsible for edly.
     """
-    edly_app_context = cache.get(EDLY_CACHE_NAME)
-    if not edly_app_context:
-        edly_app_context = {}
-        edly_app_context.update(
+    edly_context = cache.get(EDLY_CACHE_NAME)
+    if not edly_context:
+        edly_context = {}
+        panel_services_notifications_url = configuration_helpers.get_value('PANEL_NOTIFICATIONS_BASE_URL', '')
```

Comment:
Please make sure that @fayyazahmed66 is aware of setting `PANEL_NOTIFICATIONS_BASE_URL` for cloud clients.

Reply:
I will coordinate all these changes with @fayyazahmed66